executor/show_test.go

```diff
@@ -462,6 +462,31 @@ func (s *testSuite) TestShowVisibility(c *C) {
 	tk.MustExec("drop database showdatabase")
 }
 
+// ShowDatabases show the first database is INFORMATION_SCHEMA
```

Comment:
1. Remove this comment.
2. This is an integration test but not a unit test, rename this func as TestShowDatabases.
3. Add a unit test for `moveInfoSchemaToFront`.

Reply:
Thanks. is the unit test about moveInfoSchemaToFront  in the file of the show_test.go?

Reply:
yep

Reply:
but the package is executor_test, not executor, I need create one file, or can I use other files?

Reply:
ok, we can not call moveInfoSchemaToFront in package executor_test,
you can put the unit test in execuotr/pkg_test.go